language/move-lang/src/typing/translate.rs

```diff
@@ -112,6 +112,7 @@ fn check_primitive_script_arg(context: &mut Context, mloc: Loc, idx: usize, ty:
         Type_::bool(loc),
         Type_::address(loc),
         Type_::vector(loc, Type_::u8(loc)),
+        Type_::vector(loc, Type_::vector(loc, Type_::u8(loc))),
```

Comment:
not sure which version you are on but I thought @tnowacki already did all the work in the move copiler.
Todd?

Reply:
Yeah, I already did this. This change should be removed